### PR TITLE
Add Full Website

### PR DESCRIPTION
**Pull request by frempongdev**
## In this PR, I made the following changes ♻️ 
- ### Created Full Math Magician Website. 🚀 
  - Added `Home.js` and `Quote.js` components to the application.
  - Installed React-Router-Dom.
  - Added navigation to page with use of Routers.
  - Implemented Navigation `<Link>` elements to respond to page routes.
  - Updated `Home.js` with page Home page elements. ⬇️ 
  - Added Style to Home page through the use of React CSS modules.
  - Update `Quote.js` page with Quote element.
  - Added style to Quote page.
  - Adjusted Calcculator page to suit full page design.
  - Fixed Linter errors. 🧰

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+        </div>
+        <nav className={style.navBox}>
+          <ul className={style.nav}>
+            <li><Link to="/">Home</Link></li>
+            <span className={style.divider}>|</span>
+            <li><Link to="/Calculator">Calculator</Link></li>
+            <span className={style.divider}>|</span>
+            <li><Link to="/Quote">Quote</Link></li>
+          </ul>
+        </nav>
```
Comment (andersonlebon) [inline src/App.js:26]: ## [OPTIONAL]
- [x] Let's refactor a bit this code by putting it in a separate component as it contains the logic of the navigation bar.

**Review by andersonlebon — APPROVED** [review]
## Hi @frempongdev 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

## To highlight :champagne: 
- The styling is good :heavy_check_mark: 
- Well understanding of the requirements :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (frempongdev) [reply src/App.js:26]: Alright, I will work on that. 
Thank You

**Comment by frempongdev** [comment]
Thank You for taking a deep dive into my project, @andersonlebon .

**Pull request merged**